packages/gatsby/src/utils/webpack-utils.js

```diff
@@ -430,8 +430,8 @@ module.exports = async ({
    */
   plugins.extractText = options =>
     new MiniCssExtractPlugin({
-      filename: `[name].[contenthash].css`,
-      chunkFilename: `[name].[contenthash].css`,
+      filename: `../css/[name].[contenthash].css`,
```

Comment:
why is `../css/` needed?

Reply:
So, the css is generated in the stage `build-javascript` using the [MiniCssExtractPlugin](https://github.com/webpack-contrib/mini-css-extract-plugin), which grabs the output path from webpack's config `webpackConfig.output.path`.

From what I could tell, there's no way of changing the output path (only publicPath?) using `MiniCssExtractPlugin` so adjusting the filenames is a workaround. 

What's happening is the system things it's putting the CSS into the `/js` folder, but it actually goes back into the root, and creates a `/css` folder to place the CSS files in.

If there's a different way to handle this, such as creating `build-css` or a different `webpackConfig.output.path` during the time of CSS generation that would solve this workaround.